apps/system-e2e/src/tests/islandis/service-portal/smoke/notification-settings.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { test, BrowserContext, expect } from '@playwright/test'
+
+import { mNotifications } from '@island.is/service-portal/information/messages'
+import { m } from '@island.is/service-portal/core/messages'
```

Comment:
Can you please un-import these and use the strings directly? See #14283 for the rationale